etc/sources.csv

```diff
@@ -27,7 +27,6 @@ Levitskaja,1976,levitskaja1976,Istoricheskaja morfologija Chuvashskogo jazyka,co
 Andreev,1997,andreev1997,Čuvašskij Yazyk,grammar_sketch,hh:s:Andreev:Cuvasskij
 "Savelev, Alexandr",p.c. 2017,,,,
 Andreev,1961,sirotkin1961,Cuvassko-russkij slovar': okolo 25000 slov ; s prilozeniem kratkogo grammaticeskogo ocerka cuvasskogo jazyka,,mpieva:Sirotkin1961Cuvassko
-Benzing,1940,,,,

```

Comment:
As above, this reference appears in the data. What should we do with these references?

Reply:
I deleted it in the data.

Reply:
In which version of the data did you do this, i.e. what's the authoritative, editable version of the data. It does not seem to be any of the files in this repos.

Reply:
Indeed the data files are on my laptop. The sources.csv file contains 13441 rows, I'm afraid I'll get lost if I start editing the references there, so I do it language by language, in each separate .numbers sheet. The folder "raw" in this repository contains my attempts to create data files in cldf, not the raw spreadsheets with all the columns including the references. Someone else created this repository for me (I guess Christoph?) and I don't have the right to write here.

Reply:
Maybe we should put your raw spreadsheets into `raw` here? This would make them public once the cldf data goes public, too. Is this a problem?

@chrzyki what do you think?

Reply:
you should have received an invitation to collaborate on this repos, which should grant you write access.

Reply:
have just received it, thank you!

Reply:
> Maybe we should put your raw spreadsheets into `raw` here? This would make them public once the cldf data goes public, too. Is this a problem?

sounds good! I'll try to clean them up a bit and move them here. Then my changes would indeed be visible and tractable!

Reply:
We'd need to rewire the CLDF creation, but I think that's the way to go - in particular if you keep on adding to the data in the future.

Reply:
> 
> 
> Maybe we should put your raw spreadsheets into `raw` here? This would make them public once the cldf data goes public, too. Is this a problem?
> 
> @chrzyki what do you think?

Yes, I think so too: Having the raw spreadsheets as data source would make things more transparent and hopefully somewhat easier for you, @NataliiaHue.

Reply:
I've added raw spreadsheets. How do I continue working on this sources.csv file? I would like to preserve my changes, but the main branch contains the old file. Are my changes on a separate branch? Where do I find it?

Reply:
I've added raw spreadsheets and figured out how to continue working on the sources.csv in this branch.

Reply:
I'll have to merge this pull request into the main branch, because it's gotten older than the main branch in the meantime, where I have the raw data, which I would like to edit in parallel to editing the sources.csv file.